### DISCUSSION TITLE
Unable to Register Parquet Files Through Python API

### DISCUSSION THREAD

**Discussion by jayceslesar** [duckdb/duckdb#5466] [General]
### What happens?

This issue was originally exposed to me by trying to use the Ibis library with the duckdb backend but have been able to reproduce here as well.

I am unable to register Parquet files with the python API

The below code raises:
```
duckdb.InvalidInputException: Invalid Input Error: Python Object str not suitable to be registered as a view
```

### To Reproduce

```pycon
import duckdb
import pandas as pd
import numpy as np
import pyarrow as pa
import pyarrow.parquet as pq


df = pd.DataFrame({'one': [-1, np.nan, 2.5],
                   'two': ['foo', 'bar', 'baz'],
                   'three': [True, False, True]},
                   index=list('abc'))

table = pa.Table.from_pandas(df)
pq.write_table(table, 'example.parquet')

con = duckdb.connect()
con.register("test", "SELECT * FROM read_parquet('example.parquet');")
```

### OS:

Mac os

### DuckDB Version:

duckdb==0.6.0

### DuckDB Client:

Python

### Full Name:

Jayce

### Affiliation:

Beta Technologies

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by Tishj** [comment]
Is it possible you're looking for this instead?
```
from_parquet(self: duckdb.DuckDBPyConnection, file_name: str, binary_as_string: bool = False) → duckdb.DuckDBPyRelation
```
> Create a relation object from the Parquet file in file_name 

As documented here:
https://duckdb.org/docs/api/python/reference/#duckdb.DuckDBPyConnection
